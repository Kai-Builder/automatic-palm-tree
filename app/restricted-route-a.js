import Restricted from './restricted';
import React from 'react';
import {SecureRoute} from '@okta/okta-react';
// import {Route as SecureRoute} from 'react-router-dom';

const RestrictedRouteA = () => (
    <SecureRoute
        component={Restricted}
        path='/restricted-a'
    />
);

export default RestrictedRouteA;
