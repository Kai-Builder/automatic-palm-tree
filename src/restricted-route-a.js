import React from 'react';
import Restricted from './restricted';
import {
  SecureRoute as Route
} from '@okta/okta-react';

// import {Route as SecureRoute} from 'react-router-dom';

const RestrictedRouteA = () => (
  <Route
    component={Restricted}
    path='/restricted-a'
  />
);

export default RestrictedRouteA;
