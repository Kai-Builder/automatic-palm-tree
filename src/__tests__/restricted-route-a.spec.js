import React from 'react';
import {RestrictedRouteA} from '../restricted-route-a';
import {shallow} from 'enzyme';

describe('RestrictedRouteA', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<RestrictedRouteA {...props}/>);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});
