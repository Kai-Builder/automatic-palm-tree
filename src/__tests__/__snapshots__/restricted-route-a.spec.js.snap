// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RestrictedRouteA component renders 1`] = `
<withAuth(SecureRoute)
  component={[Function]}
  path="/restricted-a"
/>
`;

exports[`RestrictedRouteA component to be closed for unauthorized users 1`] = `
<withAuth(SecureRoute)
  component={[Function]}
  path="/restricted-a"
/>
`;

exports[`RestrictedRouteA path is correct 1`] = `
<withAuth(SecureRoute)
  component={[Function]}
  path="/restricted-a"
/>
`;

exports[`RestrictedRouteA route renders Restricted 1`] = `
<withAuth(SecureRoute)
  component={[Function]}
  path="/restricted-a"
/>
`;
